tests/foreman/cli/test_computeresource_osp.py

```diff
@@ -47,8 +44,15 @@ def cr_cleanup(self, cr_id):
 
     @pytest.mark.upgrade
     @pytest.mark.tier3
-    @pytest.mark.parametrize('id_type', ['id', 'name'])
-    def test_crud_and_duplicate_name(self, request, id_type):
+    @pytest.mark.parametrize(
+        'hostname',
+        [settings.osp.api_url.osp16, settings.osp.api_url.osp17],
+    )
```

Comment:
I think it'll be better to have a new parametrize fixture for this purpose which can be more flexible to run tests either on OSP16 or OSP17 or both, wdyt?

Reply:
Added your suggestion